backend/selenium_tests/page_object/people/people_details.py

```diff
@@ -0,0 +1,252 @@
+from selenium.webdriver.remote.webelement import WebElement
+
+from selenium_tests.page_object.base_components import BaseComponents
+
+
+class PeopleDetails(BaseComponents):
+    pageHeaderContainer = 'div[data-cy="page-header-container"]'
+    pageHeaderTitle = 'h5[data-cy="page-header-title"]'
+    labelFullName = 'div[data-cy="label-Full Name"]'
+    labelGivenName = 'div[data-cy="label-Given Name"]'
+    labelMiddleName = 'div[data-cy="label-Middle Name"]'
+    labelFamilyName = 'div[data-cy="label-Family Name"]'
+    labelGender = 'div[data-cy="label-Gender"]'
+    labelAge = 'div[data-cy="label-Age"]'
+    labelDateOfBirth = 'div[data-cy="label-Date of Birth"]'
+    labelEstimatedDateOfBirth = 'div[data-cy="label-Estimated Date of Birth"]'
+    labelMaritalStatus = 'div[data-cy="label-Marital Status"]'
+    labelWorkStatus = 'div[data-cy="label-Work Status"]'
+    labelPregnant = 'div[data-cy="label-Pregnant"]'
+    labelRole = 'div[data-cy="label-Role"]'
```

Comment:
```
class YourClass:
    def __init__(self):
        self.labels = {
            "pageHeaderContainer": 'div[data-cy="page-header-container"]',
            "pageHeaderTitle": 'h5[data-cy="page-header-title"]',
            "labelFullName": 'div[data-cy="label-Full Name"]',
            "labelGivenName": 'div[data-cy="label-Given Name"]',
            # ... add the rest of your labels here ...
            "labelUserName": 'div[data-cy="label-User name"]',
        }

    def get_label(self, name: str) -> str:
        return self.labels[name]
```
        

        Similar approach can be done here. Not necessarily in this PR,
        but maybe something to think about in the future.